### PR TITLE
feat(note-taking): add support for zettelkasten with zk-nvim

### PR DESCRIPTION
**Pull request by gacallea**
## 📑 Description

Adding support for zk-nvim.

## ℹ Additional Information

All about Zettelkasten here: https://zettelkasten.de/posts/overview/

### CODE REVIEW

File: lua/astrocommunity/note-taking/zk-nvim/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Adds `zk-nvim`
+Adds `zk` language server
+
+## NOTE
+
+This plugin will:
+
+- setup and start the LSP server for you, do not call require("lspconfig").zk.setup().
+- automatically attach buffers in a zk notebook that match the given filetypes
```
Comment (Uzaaft) [inline lua/astrocommunity/note-taking/zk-nvim/README.md:23]: This should be:
```suggestion
# zk-nvim

Neovim extension for the [zk](https://github.com/mickael-menu/zk) plain text note-taking assistant.

**Repository:** <https://github.com/mickael-menu/zk-nvim>

> :warning: This plugin will setup and start the LSP server for you, do *not* call `require("lspconfig").zk.setup()`.


```

**Review by Uzaaft — CHANGES_REQUESTED** [review]
plugin spec seems alright, please take a look at the changes I suggest for the docs

**Review by Uzaaft — APPROVED** [review]

File: lua/astrocommunity/note-taking/zk-nvim/init.lua [hunk 2]
```diff
@@ -0,0 +1,17 @@
+local utils = require "astronvim.utils"
+
+return {
+  {
+    "mickael-menu/zk-nvim",
+    config = function()
+      require("zk").setup {
+        picker = "telescope",
+      }
+    end,
```
Comment (Uzaaft) [inline lua/astrocommunity/note-taking/zk-nvim/init.lua:10]:
```suggestion
    opts = {
    	picker = "telescope"
    },
 
```

File: lua/astrocommunity/note-taking/zk-nvim/init.lua [hunk 3]
```diff
@@ -0,0 +1,16 @@
+local utils = require "astronvim.utils"
+
+return {
+  {
+    "mickael-menu/zk-nvim",
+    opts = {
+    	picker = "telescope"
+    },
+
```
Comment (Uzaaft) [inline lua/astrocommunity/note-taking/zk-nvim/init.lua:9]:
```suggestion
```

**Review by Uzaaft — APPROVED** [review]

**Comment by gacallea** [comment]
@Uzaaft sorry about the opts = {} missing and the style mishaps.

**Comment by Uzaaft** [comment]
> @Uzaaft sorry about the opts = {} missing and the style mishaps.

I believe I fucked up the style 🤣

**Pull request merged**